packages/nimble/R/genCpp_sizeProcessing.R

```diff
@@ -207,6 +214,22 @@ sizeProxyForDebugging <- function(code, symTab, typeEnv) {
     return(ans)
 }
 
+## This is used by nimbleExternalCall
```

Comment:
nit: Punctuate sentences in comments. As yourself "Is this how I would write in a preprint"?